core/src/dird/ua_configure.cc

```diff
@@ -135,17 +136,31 @@ static inline bool config_add_directive(UaContext* ua,
 {
   PoolMem temp(PM_MESSAGE);
   ResourceItem* item = NULL;
+  std::string format("%-*s%s = %s\n");
+
+  const std::vector<int> quotable_types{
```

Comment:
Why not use a std::array here? The size is fixed and will not be altered so std::array is the better option I guess.